packages/webapp/src/components/Navigation/TopMenu/TopMenu.jsx

```diff
@@ -131,18 +135,39 @@ const TopMenu = ({ history, isMobile, showNavigation, onClickBurger }) => {
   });
 
   const floaterMenu = (
-    <Menu
-      id="profile-menu"
-      anchorEl={profileIconRef.current}
+    <Popper
       open={openMenu}
-      onClose={closeMenu}
-      MenuListProps={{
-        'aria-labelledby': 'profile-navigation-button',
+      anchorEl={profileIconRef.current}
+      role={undefined}
+      placement="bottom-start"
+      transition
+      disablePortal
+      sx={{
+        top: '8px !important',
       }}
-      classes={{ list: styles.menuList, root: styles.menuRoot }}
     >
-      {menuItems}
-    </Menu>
+      {({ TransitionProps, placement }) => (
+        <Grow
+          {...TransitionProps}
+          style={{
+            transformOrigin: placement === 'bottom-start' ? 'left top' : 'left bottom',
```

Comment:
If this is changed to `right top`, the behaviour looks the same to me.

Reply:
nice touch!